pkg/task/config.go

```diff
@@ -0,0 +1,19 @@
+package task
+
+type runConfig struct {
+	runUpdateCheck bool
+}
+
+func newRunConfig() *runConfig {
+	return &runConfig{
+		runUpdateCheck: true,
+	}
+}
+
+type TaskRunConfig func(o *runConfig)
```

Comment:
`RunOption`

Reply:
I think our pattern is always - use an `Option` to set a `Configg`

Reply:
done